docs/REST API/rest-api.md

```diff
@@ -2511,6 +2511,10 @@ Returns the information of the specified host.
         "coordinates": [40.6799, -74.0028]
       }
     },
+    "maintenance_window": {
+      "starts_at": "2024-06-18T13:27:18−07:00 UTC−07:00",
```

Comment:
@noahtalerman thanks for checking! `starts_at` actually doesn't include the "UTC-..." part, so this should just be `"2024-06-18T13:27:18−07:00"`

Reply:
Also, `"America/New_York"`'s UTC offset is either 4 or 5depending on DST. Currently it's -4, so we'll want `"2024-06-18T13:27:18−04:00"`

Reply:
Thanks Jacob!

I updated the docs to `"2024-06-18T13:27:18−04:00"`